src/utils/setup/handle-create-page.js

```diff
@@ -35,6 +35,11 @@ const memoizedStitchRequest = memoizerific(10)(requestStitch);
 
 const getAllArticles = memoizerific(1)(async () => {
     const documents = await memoizedStitchRequest('fetchDevhubMetadata', {});
+    // We don't need all of the article contents for preview
+    documents.forEach(doc => {
```

Comment:
The above Realm function is part of the snooty app which gives us everything for every article. Ideally we could project out AST and source Realm-side since we don't need them which would also cut down data going over the network